doc/rules.md

```diff
@@ -0,0 +1,72 @@
+### rules
+
+
+capa uses a collection of rules to identify capabilities within a program.
+The [github.com/mandiant/capa-rules](https://github.com/mandiant/capa-rules) repository contains hundreds of standard library rules that are distributed with capa.
+
+When you download a standalone version of capa, this standard library is embedded within the executable and capa will use these rules by default:
+
+```console
+$ capa suspicious.exe
+```
+
+However, you may want to modify the rules for a variety of reasons:
+
+  - develop new rules to find behaviors, and/or
+  - tweak existing rules to reduce false positives, and/or
+  - collect a private selection of rules not shared publicly.
+
+Or, you may want to use capa as a Python library within another application.
+
+In these scenarios, you must provide the rule set to capa as a directory on your file system. Do this using the `-r`/`--rules` parameter:
+
+```console
+$ capa    --rules /local/path/to/rules    suspicious.exe
+```
+
+You can collect the standard set of rules in two ways:
+
+  - [download from the Github releases page](#download-release-archive), or
+  - [clone from Github](#clone-with-git).
+
+Note that you must use match the rules major version with the capa major version,
+i.e., use `v1` rules with `v1` of capa.
+This is so that new versions of capa can update rule syntax, such as by adding new fields and logic.
+
+Otherwise, using rules with a mismatched version of capa may lead to errors like:
+
+```
+$ capa   --rules /path/to/mismatched/rules    suspicious.exe
+ERROR:lint:invalid rule: injection.yml: invalid rule: unexpected statement: instruction
+```                
+
+You can check the version of capa you're currently using like this:
+                         
+```console
+$ capa --version
+capa 3.0.3
+```
+
+#### download release archive
+
+The releases page is [here](https://github.com/mandiant/capa-rules/tags/).
+Find the most recent release corresponding to your major version of capa and download the ZIP archive.
+Here are some quick links:
+  - v1: [v1.6.3](https://github.com/mandiant/capa-rules/releases/tag/v1.6.1)
+  - v2: [v2.0.0](https://github.com/mandiant/capa-rules/releases/tag/v2.0.0)
+  - v3: [v3.0.3](https://github.com/mandiant/capa-rules/releases/tag/v3.0.3)
```

Comment:
```suggestion
  - v1: [v1](https://github.com/mandiant/capa-rules/releases/tag/v1)
  - v2: [v2](https://github.com/mandiant/capa-rules/releases/tag/v2)
  - v3: [v3](https://github.com/mandiant/capa-rules/releases/tag/v3)
```

Reply:
TODO: create these manually, in the future will be handled by updated Action (see above)

Reply:
you read my mind: https://github.com/mandiant/capa/pull/1006#discussion_r866953156 😆

Reply:
```console
$ git tag --format="%(color:bold cyan)== %(refname:short) ==%(if)%(object)%(then)%0aTag Hash: %(objectname)%0aTag Date: %(taggerdate:iso-local)%0a  Commit: %(object) %0a%0a%(contents)%(else)%0a(lightweight tag)%0a  Commit: %(objectname)%(end)%0a"

== v1 ==
(lightweight tag)
  Commit: 93ea28dd325c2db5e080272063c4dc12276c49cd

...

== v1.6.1 ==
(lightweight tag)
  Commit: 93ea28dd325c2db5e080272063c4dc12276c49cd

== v2 ==

(lightweight tag)
  Commit: 00490c1f68b422acd3488fd7b700a5d1d754fb41

== v2.0.0 ==
Tag Hash: 118cc3126d87d4d8b943e5eefd19dc1b61c40ce6
Tag Date: 2021-07-19 18:38:38 +0200
  Commit: 00490c1f68b422acd3488fd7b700a5d1d754fb41

https://github.com/fireeye/capa/releases/v2.0.0


== v3 ==
(lightweight tag)
  Commit: 27123103e241d892e814e810239e3a06a054b65a

...


== v3.2.0 ==
Tag Hash: 81e9559fa16ca092aadc6f8f530e05b60224157e
Tag Date: 2022-03-03 20:03:58 +0100
  Commit: 27123103e241d892e814e810239e3a06a054b65a

https://github.com/mandiant/capa/releases/v3.2.0


```

Reply:
TODO: standardize annotated tag format and messages